src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -190,13 +203,15 @@ public void stop() {
     }
 
     public void coast() {
-        driveMotor.setIdleMode(IdleMode.kCoast);
-        turningMotor.setIdleMode(IdleMode.kCoast);
+        int errorCode = driveMotor.setIdleMode(IdleMode.kCoast).value;
+        errorCode += turningMotor.setIdleMode(IdleMode.kCoast).value;
```

Comment:
Check drive motor and turning motor separately

Reply:
done, motors are now checked separately